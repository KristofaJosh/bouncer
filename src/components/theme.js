import * as colors from "../components/Colors/color";
const theme = {
  colors,
};
export default theme;