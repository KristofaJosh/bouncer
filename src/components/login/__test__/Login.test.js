import React from "react";
import renderer from "react-test-renderer";
import { ThemeProvider } from "styled-components";
import { BrowserRouter } from "react-router-dom";

import SignIn from "../";
import theme from "../../theme";

it("renders correctly when component is called", () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={theme}>
        <BrowserRouter>
          <SignIn />
        </BrowserRouter>
      </ThemeProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
