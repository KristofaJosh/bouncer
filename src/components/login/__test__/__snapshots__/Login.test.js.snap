// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when component is called 1`] = `
<div
  className="sc-bdVaJa ixFDBh"
>
  <div
    className="loginImageContainer"
  >
    <img
      alt="Login"
      src="loginImage.png"
    />
  </div>
  <div
    className="loginForm"
  >
    <div
      className="header"
    >
      <a
        className="sc-bwzfXH bPPCVC active pb-2"
        href="/login"
        onClick={[Function]}
      >
        Sign in
      </a>
      <a
        className="sc-bwzfXH bPPCVC"
        href="/register"
        onClick={[Function]}
      >
        Register
      </a>
    </div>
    <div
      className="form"
    >
      <form>
        <div
          className="sc-ifAKCX evpAft"
        >
          <label
            className="sc-bxivhb bvzMcK"
            htmlFor="input"
          >
             
            Email
             
          </label>
          <input
            aria-label="input"
            aria-required="true"
            className="sc-htpNat ieoNZF"
            label="Email"
            name="email"
            placeholder="Enter Email Here"
            type="email"
          />
        </div>
        <div
          className="sc-ifAKCX evpAft"
        >
          <label
            className="sc-bxivhb bvzMcK"
            htmlFor="input"
          >
             
            Password
             
          </label>
          <input
            aria-label="input"
            aria-required="true"
            className="sc-htpNat ieoNZF"
            label="Password"
            name="password"
            placeholder="Enter Password Here"
            type="password"
          />
        </div>
        <button
          className="sc-EHOje hKRBx"
        >
          Login
        </button>
      </form>
    </div>
    <div
      className="footer"
    >
      <a
        className="sc-bwzfXH bPPCVC"
        href="/forgot-password"
        onClick={[Function]}
      >
        Forgot password
      </a>
    </div>
  </div>
</div>
`;
